maxim/tests/test_fireworks.py

```
@@ -0,0 +1,495 @@
+            delattr(Maxim, "_instance")
+        
+        if not fireworksApiKey:
+            raise ValueError("FIREWORKS_API_KEY environment variable is not set")
+
+        self.logger = Maxim().logger()
+        
+        # Initialize Fireworks LLM client with instrumentation
+        self.llm = LLM(
+            model="llama4-maverick-instruct-basic", 
+            deployment_type="serverless", 
+            api_key=fireworksApiKey
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)